fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceDAOImpl.java

```diff
@@ -459,40 +461,19 @@ public void addResourceTypeCacheCandidate(String resourceType, Integer resourceT
 
     }
 
-    @Override
-    public Resource insert(Resource resource, List<ExtractedParameterValue> parameters, ParameterDAO parameterDao)
-            throws FHIRPersistenceException {
-        final String METHODNAME = "insert(Resource, List<ExtractedParameterValue>";
-        log.entering(CLASSNAME, METHODNAME);
+    public boolean isPostgreSqlDatabase() throws FHIRPersistenceDBConnectException, SQLException {
```

Comment:
This feels like it should go into the another method.

Reply:
good catch, I forgot to remove it. it should have been dropped in my refactoring. done